apps/ledger-live-desktop/tests/specs/speculos/accountSpeculos.spec.ts

```diff
@@ -0,0 +1,91 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { PortfolioPage } from "../../models/PortfolioPage";
+import { AddAccountModal } from "../../models/AddAccountModal";
+import { Layout } from "../../models/Layout";
+import { AccountPage } from "../../models/AccountPage";
+import { AccountsPage } from "../../models/AccountsPage";
+import { Device, specs, startSpeculos, stopSpeculos } from "../../utils/speculos";
+import { Currency } from "../../enum/Currency";
+
+test.use({ userdata: "skip-onboarding" });
+const currencies: Currency[] = [
+  Currency.BTC,
+  Currency.tBTC,
+  Currency.ETH,
+  Currency.tETH,
+  Currency.SOL,
+  Currency.DOT,
+  Currency.TRX,
+  Currency.XRP,
+];
+
+let device: Device | undefined;
+
+test.afterEach(async () => {
+  await stopSpeculos(device);
+});
+
+test.describe.parallel("Accounts @smoke", () => {
+  for (const currency of currencies) {
+    let firstAccountName = "NO ACCOUNT NAME YET";
+    let accountsListBeforeRemove: (string | null)[];
+    test(`[${currency.uiName}] Add account`, async ({ page }) => {
+      const portfolioPage = new PortfolioPage(page);
+      const addAccountModal = new AddAccountModal(page);
+      const layout = new Layout(page);
+      const accountsPage = new AccountsPage(page);
+      const accountPage = new AccountPage(page);
+      device = await startSpeculos(test.name, specs[currency.deviceLabel.replace(/ /g, "_")]);
```

Comment:
on our peer programming , we moved the management of speculos on the "common.ts" side, which also will be more robust to not have to deal with this on test side so it would worth integrating.
in fact i already see in this PR two similar tests that could benefit this factorisation we did at the shared common.ts level.